### ISSUE TITLE
Wabbajack does not detect correct resolution

### ISSUE THREAD

**Issue by Denial7** [wabbajack-tools/wabbajack#545]
**Describe the bug**
Upon installation of any list, the iSizeH and W in the Skyrim/FalloutPrefs.ini are set to a much lower value than my actual screen resolution is (1920x1080) and so far as I remember I've had to go correct this for all (SSE, FO4, FNV, TESIV) lists I've downloaded, and seen others in support channels with the same issue.

**Additional context**
Windows 10, Laptop - when I look at the advanced display settings it shows up has having an Intel UHD Graphics 630, and the monitor shows up as a 'Generic PnP Monitor'. Let me know if I can provide more helpful specifics.

**Logs**
N/A

**Solution**

**Comment by halgari** [comment]
@Denial7 what does it detect? If I have the exact bad dimensions it might help me figure out where the setting is going wrong. E.g. if the number is exactly 1/2 what is expected it would indicate that something with screen scaling is going on.

**Comment by halgari** [comment]
Sounds like the code we use can return the wrong value for non-standard DPI screens (https://stackoverflow.com/questions/2630392/getsystemmetrics-returns-wrong-value-for-sm-cxscreen. I'm not sure I want to add DPI awareness this late in the dev cycle, but there's certainly some way we can get a fix in for this for this issue.

**Comment by Denial7** [comment]
> @Denial7 what does it detect? If I have the exact bad dimensions it might help me figure out where the setting is going wrong. E.g. if the number is exactly 1/2 what is expected it would indicate that something with screen scaling is going on.

Apologies, tried to find this last night but when I reinstalled Lotus it had the resolution correct, maybe it it didn't change the ini because I already had it. I'll try a fresh install of a list today and see if I can get this value, it's something like 500x800 though.

**Comment by Denial7** [comment]
Okay so I installed the new Oblivion list WOOL and the ini settings were correct (h=1080, w=1920) in the Oblivion.ini which confused me. So I deleted the .ini files for Trawz's list and reinstalled that, the resolution in the SkyrimPrefs.ini is H=864 W=1536.

**Comment by halgari** [comment]
I've replicated this on my own machine, and it has to do with UI scaling. When I set scaling to 100% on my main display WJ outputs the correct resolution. When I set scaling to 125% it's wrong by the expected amount. 

I'm digging into how to fix this, but unfortunately Windows is really wonky in how it looks up this information. WJ itself needs to have the resolution scaled, but I have yet to find a backdoor to get the actual resolution in an app that's already told Windows it can handle scaling.
